yarn.lock

```diff
@@ -6,50 +6,7 @@
   version "1.0.0"
   resolved "https://registry.yarnpkg.com/@allmarkedup/fang/-/fang-1.0.0.tgz#20aa2ea365c57b0a28e6b563613270f5630d6883"
 
-"@frctl/fractal@^1.0.0", "@frctl/fractal@^1.1.0-alpha.0":
```

Comment:
Weird... You didn't change any dependency but there are changes. Check if you have the latest version of Yarn and reinstall dependencies locally.

Reply:
wierd, I have the latest version of yarn and I reinstall dependencies but nothing new in yarn.lock. When I've been working on this task I have tried some npm packages, maybe I commited it by mistake, should I fix it manually?